spec/calculator.spec.js

```diff
@@ -21,281 +21,281 @@
     });
 
     it('It should be able to perform a simple add operation', () => {
-      instance.addNumber(3)();
+      instance.addNumber(3);
```

Comment:
Remove all the extra () after addNumber(x) because addNumber is a function now, it no longer returns a function